unittest2.nim

```diff
@@ -226,7 +226,7 @@ type
   JUnitTest = object
     name: string
     result: TestResult
-    error: (seq[string], string)
+    error: tuple[a: seq[string], b: seq[string]]
```

Comment:
still needs better names for `a` and `b`

Reply:
and use those names where relevant

Reply:
done